contents/blog/features-sell.md

```diff
@@ -0,0 +1,26 @@
+---
+date: 2022-10-20
+title: "Is this the death of benefits selling?"
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: ["CEO diaries", "Inside PostHog"]
+author: ["james-hawkins"]
+featuredImage: ../images/blog/posthog-ceo-diary-blog.png
+featuredImageType: full
+---
+
+Features tell, benefits sell. Well, they used to.
+
+Marketing lore tells you that you must focus on the _why_, not the what. "No one buys a CRM, they buy a way to increase their revenue" - the sort of thing you hear at a TED Talk.
```

Comment:
```suggestion
Marketing lore tells you that you must focus on the _why_, not the what. "No one buys a CRM, they buy a way to increase their revenue".
```

Reply:
removed simon sinek reference as it's exactly the same concept, but marketers don't usually quote him